cedar-policy-cli/src/lib.rs

```diff
@@ -588,6 +639,41 @@ pub fn format_policies(args: &FormatArgs) -> CedarExitCode {
     }
 }
 
+fn translate_to_natural(json_src: impl AsRef<str>) -> Result<String> {
+    let fragment = SchemaFragment::from_str(json_src.as_ref())?;
+    let output = fragment.as_natural()?;
+    Ok(output)
+}
+
+fn translate_to_json(natural_src: impl AsRef<str>) -> Result<String> {
+    let (fragment, warnings) = SchemaFragment::from_str_natural(natural_src.as_ref())?;
+    for warning in warnings {
+        eprintln!("{warning}");
```

Comment:
if these are `miette::Diagnostic` then we can make a `miette::Report` containing all the warnings, and print that (with Debug, because that's how you get the pretty miette output; the Display is still always just the main error message)